recipes/fftw/all/conanfile.py

```diff
@@ -107,29 +101,21 @@ def on_off(value):
             return "ON" if value else 'OFF'
 
         apply_conandata_patches(self)
-        for self._current_precision in self._all_precisions:
+        for current_precision in self._all_precisions:
             cmake = CMake(self)
             variables = {
-                "ENABLE_FLOAT": on_off(self._current_precision == SINGLE),
-                "ENABLE_LONG_DOUBLE": on_off(self._current_precision == LONGDOUBLE),
-                "ENABLE_QUAD_PRECISION": on_off(self._current_precision == QUAD)
+                "ENABLE_FLOAT": on_off(current_precision == SINGLE),
+                "ENABLE_LONG_DOUBLE": on_off(current_precision == LONGDOUBLE),
+                "ENABLE_QUAD_PRECISION": on_off(current_precision == QUAD)
             }
             cmake.configure(variables=variables)
             cmake.build()
-
-        # Potentially avoid side effects due to build_folder property tweak.
-        self._current_precision = None
+            cmake.install()
```

Comment:
I think the proposed approach makes sense: files are added to the install folder (the libraries), and every other file is just overwritten with exactly the same contents. This is equivalent to the way it's done in CMake to support multi-configuration (Release/Debug) in the same install tree: configure twice, build twice, install twice to same location.

Was wondering if it would be worth passing `--fresh` to CMake configure, or clearing the build folder ahead of starting, but from what I can see, we wouldn't benefit much, as the result is the same.